scripts/hash_password

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+import argparse
+import bcrypt
+import getpass
+
+bcrypt_rounds=12
```

Comment:
the number of rounds is recorded in the hash (`$2a$12$xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx` -> salt is 2a, # of rounds is 12) so synapse will be happy with the generated hash either way.